cli/hzn.go

```diff
@@ -383,9 +383,9 @@ Environment Variables:
 	policyCmd := app.Command("policy", msgPrinter.Sprintf("List and manage policy for this Horizon edge node."))
 	policyListCmd := policyCmd.Command("list", msgPrinter.Sprintf("Display this edge node's policy."))
 	policyNewCmd := policyCmd.Command("new", msgPrinter.Sprintf("Display an empty policy template that can be filled in."))
-	policyUpdateCmd := policyCmd.Command("update", msgPrinter.Sprintf("Update the node's policy. The node's built-in properties will be automatically added if the input policy does not contain them."))
+	policyUpdateCmd := policyCmd.Command("update", msgPrinter.Sprintf("Create or replace the node's policy. The node's built-in properties will be automatically added if the input policy does not contain them."))
 	policyUpdateInputFile := policyUpdateCmd.Flag("input-file", msgPrinter.Sprintf("The JSON input file name containing the node policy.")).Short('f').Required().String()
-	policyPatchCmd := policyCmd.Command("patch", msgPrinter.Sprintf("Add or update node policy properties or overwrite the node policy constraint expression."))
+	policyPatchCmd := policyCmd.Command("patch", msgPrinter.Sprintf("(Deprecated) Update node policy properties or overwrite the node policy constraint expression."))
```

Comment:
Please change the message to:
"(DEPRECATED) This command is deprecated. Please use 'hzn policy update' to update the node policy. This command is used to update either the node policy properties or the constraints, but not both."

Reply:
done